models/attendance.js

```diff
@@ -0,0 +1,15 @@
+import connector from "./databaseUtil";
+import Infrastructure from "./infrastructure";
+connector.set("debug", true);
+
+const attendanceSchema = {
+  date: { type: Date, required: true },
+  time: { type: String, required: true },
+  class: Infrastructure,
+};
+
+const Attendance = connector.model("Attendance", attendanceSchema);
```

Comment:
can you rename this to from Attendance to attendanceModel? replace the same in export default as well.

Reply:
Let this be as it is for now